README.md

```diff
@@ -111,7 +111,7 @@ Merged reports are reports that combine statistics of code coverage by test task
 At the same time, for each project, its configuration of instrumentation and special filters (non-class filters) is applied.
 
 For every Gradle project participating in the merged report, both Kover plugin should be applied
-and Coverage Engine types and versions should be identical.
+and Coverage Tool types and versions should be identical.
```

Comment:
Could you please elaborate on the choice "Engine -> Tool"? 
Engine seems far more natural for me, but I might be mistaken

Reply:
In fact, the agent and other libraries are something third-party to the Kover Gradle Plugin (KGP). 
If we perceive them as something external, third-party, something auxiliary, then it began to seem to me that the word "engine" is more associated with something internal, its own code. In addition, the plugin itself may have some kind of its own "engine" entity inside.

When I was trying to find an alternative, I noticed that JaCoCo is most often called a tool (or toolkit), so I thought it was quite an appropriate name (e.g. https://www.eclemma.org/ , in [teamcity]( https://www.jetbrains.com/help/teamcity/jacoco.html) or `toolVersion` in [Gradle](https://docs.gradle.org/current/userguide/jacoco_plugin.html#sec:configuring_the_jacoco_plugin) ).

In addition, if we add a CLI over the agent and start delivering it separately without KGP, then "Kover Coverage Tool" is a more telling name than "Kover Coverage Engine".

Reply:
Thanks, this reasoning works for me